ddtrace/internal/telemetry/data/integration.py

```diff
@@ -0,0 +1,21 @@
+from typing import TypedDict
+
+
+# Stores information about modules we attempt to instrument
+Integration = TypedDict(
+    "Integration",
+    {"name": str, "version": str, "enabled": bool, "auto_enabled": bool, "compatible": str, "error": str},
+)
+
+
+def create_integration(name, version="", enabled=True, auto_enabled=True, compatible="", error=""):
```

Comment:
Currently I am using `create_integration("module_name")` to create an integration object in _monkey.patch(). I am only sending integrations that are enabled, enabled by default, are compatible with our library (we can set this field to "yes" or an empty string it doesn't matter), and there is no error. So I thought it would be cleaner to just create a helper that hides away most of the default values. Here's an example of where it's used:  [_monkey.py#L132](https://github.com/DataDog/dd-trace-py/blob/bf38df143589ff7e2077570611e07abb3a53c0be/ddtrace/_monkey.py#L132)